lib/ansible/plugins/filter/mathstuff.py

```diff
@@ -123,14 +129,28 @@ def union(environment, a, b):
     return c
 
 
-def min(a):
-    _min = __builtins__.get('min')
-    return _min(a)
+@environmentfilter
+def min(environment, a, **kwargs):
+    if HAS_MIN_MAX:
+        return do_min(environment, a, **kwargs)
+    else:
+        if kwargs:
+            raise AnsibleFilterError("Ansible's min filter does not support any keyword arguments. "
+                                     "You need a newer version of Jinja2 that provides their version of the filter.")
```

Comment:
We should specify what jinja2 version this is.  I believe it is Jinja2 2.10

Reply:
@sivel, I updated the error message to specify the version. Good idea!